template/java8/model/src/test/java/RequestTest.java

```diff
@@ -0,0 +1,23 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.openfaas.model.Request;
+public class RequestTest {
+
+	@Test
+	public void testRequestParametersGetSet()
```

Comment:
I'd like to see:

- a test which has multiple parameters
- a test which has no parameters at all
- a test which has an empty parameter